mlflow/store/artifact/databricks_sdk_models_artifact_repo.py

```diff
@@ -0,0 +1,86 @@
+import posixpath
+
+from mlflow.entities import FileInfo
+from mlflow.store.artifact.cloud_artifact_repo import CloudArtifactRepository
+
+DOWNLOAD_CHUNK_SIZE = 1024*1024*1024
+
+def _get_databricks_workspace_client():
+    from databricks.sdk import WorkspaceClient
+
+    return WorkspaceClient()
+
+
+class DatabricksSDKModelsArtifactRepository(CloudArtifactRepository):
+    """
+    Stores and retrieves model artifacts via Databricks SDK, agnostic to the underlying cloud
+    that stores the model artifacts.
+    """
+
+    def __init__(self, model_name, model_version):
+        self.model_name = model_name
+        self.model_version = model_version
+        self.model_base_path = f"/Models/{model_name.replace('.', '/')}/{model_version}"
+        self.w = _get_databricks_workspace_client()
```

Comment:
```suggestion
        self.client = _get_databricks_workspace_client()
```

can we rename this attribute?